client/src/components/PicturesView.jsx

```diff
@@ -23,15 +24,11 @@ class PicturesView extends React.Component {
   render() {
     return (
       <div>
-        <div style={{left: 0}}>
-          <PictureList 
-            pictureURLs={this.state.pictures} 
-            onHover={this.onIconHoverHandler.bind(this)}
-            idSelected={this.state.idSelected} />
-        </div>
-        <div>
-          <PictureDisplay pictureURL={this.state.current} numPictures={this.state.pictures.length} />
-        </div>
+        <PictureList 
+          pictureURLs={this.state.pictures} 
+          onHover={this.onIconHoverHandler.bind(this)}
```

Comment:
As you mentioned in my review, you might want to put the bindings within the constructor, so the function that you pass looks cleaner

Reply:
Yep, you're right.